DESIGN-NOTES.md

```diff
@@ -110,15 +110,15 @@ Many profiler consumers will need callbacks when interesting events happen in th
         + Method Info
         * IL Graph
 4. OnClassLoad
-    * Intellitrace uses this callback which occurs every time a new class loads to update the type system. This callback MUST be optional as it incurs high overhead
+    * IntelliTrace uses this callback which occurs every time a new class loads to update the type system. This callback MUST be optional as it incurs high overhead
     * Parameters:
         * Method Info
         * Class Info?
         * IMetadataImport
 5. Function Enter / Function Exit
-    * These callbacks are issued every time a function is called and every time a function returns. Intellitrace and Production BPs don’t use these, but Tofino does. Note that these also incur large runtime overhead and use a naked calling convention making it difficult to multiplex
+    * These callbacks are issued every time a function is called and every time a function returns. IntelliTrace and Production BPs don’t use these, but Tofino does. Note that these also incur large runtime overhead and use a naked calling convention making it difficult to multiplex
```

Comment:
>Production BPs [](start = 117, length = 14)

Also remove references to Production BPs. These are implementation details of that product, not CLRInstrumentationEngine.